code/_onclick/click_mob.dm

```diff
@@ -195,7 +195,7 @@
 
 /mob/living/UnarmedAttack(var/atom/A, var/proximity_flag)
 
-	if(!ticker)
```

Comment:
just remove this check, its a tiny waste of performance, nobody except admins is around when theres no ticker anyway

Reply:
Ready